### PR TITLE
Update spring version and cbioportal version

### PR DESCRIPTION
**Pull request by ao508**
Signed-off-by: Angelica Ochoa <15623749+ao508@users.noreply.github.com>

### CODE REVIEW

**Comment by ao508** [comment]
@inodb Build is breaking because the annotationPipeline child module isn't recognizing its sibling module that's specified in its pom.... ideas? I'm thinking I'm just having tunnel vision and I'm just missing something obvious

Some other changes:
- added maven compiler version (not having this has burned us before)

**Comment by inodb** [comment]
@ao508 maybe adding the spring boot configuration linked below will fix it? I vaguely recall seeing these errors. It seems limited to the annotator module

https://github.com/genome-nexus/genome-nexus-annotation-pipeline/pull/165/files#diff-c159c7b7adbb857d104c756c1b926200cb868aeaf18812adfc160048a336df83R48-R55

File: annotationPipeline/src/main/resources/application.properties.EXAMPLE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+spring.main.web-application-type=none
```
Comment (inodb) [inline annotationPipeline/src/main/resources/application.properties.EXAMPLE:1]: I still think we should handle this differently. Not the best user experience to have their pipeline crash after an upgrade. But ok to leave as is. Filed a separate issue for this: https://github.com/genome-nexus/genome-nexus-annotation-pipeline/issues/168

Comment (ao508) [reply annotationPipeline/src/main/resources/application.properties.EXAMPLE:1]: Will be spending more time on the build issue today so I'll take a look at this as well

**Pull request closed without merging**